Jenkinsfile

```diff
@@ -1 +1,13 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
\ No newline at end of file
+buildPlugin(configurations: [
+          // Sanity on the minimum required Jenkins version
+         [ platform: "linux", jdk: "8", jenkins: '2.190.3' ],
+
+          // Sanity on the recent Jenkins version
+         [ platform: "linux", jdk: "8", jenkins: null ],
+
+         // + Java 11
+         [ platform: "linux", jdk: "11", jenkins: null, javaLevel: "8"],
```

Comment:
You do not actually need to specify `javaLevel`. This is specified in https://github.com/jenkinsci/electricflow-plugin/blob/2e6b4a58cc763d722780359cbca40ff0a33ec55e/pom.xml#L18 I am not sure why this property even exists. Maybe dates back to some ancient parent POM version where `java.level` was defaulted.